package.json

```diff
@@ -16,12 +16,14 @@
   "dependencies": {
```

Comment:
running `npm run serve` gets this error.
```
ERROR in /Users/faraonc/WebstormProjects/hwsc-frontend/node_modules/@hwsc/hwsc-api-blocks-lisa-test/protobuf/hwsc-app-gateway-svc/app/hwsc-app-gateway-svc_pb.d.ts
1:23 Could not find a declaration file for module 'google-protobuf'. '/Users/faraonc/WebstormProjects/hwsc-frontend/node_modules/google-protobuf/google-protobuf.js' implicitly has an 'any' type.
  Try `npm install @types/google-protobuf` if it exists or add a new declaration (.d.ts) file containing `declare module 'google-protobuf';`
  > 1 | import * as jspb from "google-protobuf"
      |                       ^
    2 | 
    3 | import * as protobuf_hwsc$user$svc_user_hwsc$user$svc_pb from '../../../protobuf/hwsc-user-svc/user/hwsc-user-svc_pb';
    4 | import * as protobuf_lib_user_pb from '../../../protobuf/lib/user_pb';
```

Do we need ` npm install @types/google-protobuf`?

Reply:
`npm install @types/google-protobuf` seems to fix the issue but still
```
faraonc@~/WebstormProjects/hwsc-frontend (feature/vuex-gateway-init)$ npm run serve

> hwsc-frontend@0.0.1 serve /Users/faraonc/WebstormProjects/hwsc-frontend
> vue-cli-service serve

 INFO  Starting development server...
Starting type checking service...
Using 1 worker with 2048MB memory limit
 98% after emitting CopyPlugin                                                    

 WARNING  Compiled with 2 warnings                                                                                                                                                                                                                               12:55:36 AM

Module Warning (from ./node_modules/eslint-loader/index.js):
error: Line 11 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:11:1:
   9 |     <h3>Installed CLI Plugins</h3>
  10 |     <ul>
> 11 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel" target="_blank" rel="noopener">babel</a></li>
     | ^
  12 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-typescript" target="_blank" rel="noopener">typescript</a></li>
  13 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
  14 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-unit-jest" target="_blank" rel="noopener">unit-jest</a></li>


error: Line 12 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:12:1:
  10 |     <ul>
  11 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel" target="_blank" rel="noopener">babel</a></li>
> 12 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-typescript" target="_blank" rel="noopener">typescript</a></li>
     | ^
  13 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
  14 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-unit-jest" target="_blank" rel="noopener">unit-jest</a></li>
  15 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-cypress" target="_blank" rel="noopener">e2e-cypress</a></li>


error: Line 13 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:13:1:
  11 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel" target="_blank" rel="noopener">babel</a></li>
  12 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-typescript" target="_blank" rel="noopener">typescript</a></li>
> 13 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
     | ^
  14 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-unit-jest" target="_blank" rel="noopener">unit-jest</a></li>
  15 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-cypress" target="_blank" rel="noopener">e2e-cypress</a></li>
  16 |     </ul>


error: Line 14 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:14:1:
  12 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-typescript" target="_blank" rel="noopener">typescript</a></li>
  13 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
> 14 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-unit-jest" target="_blank" rel="noopener">unit-jest</a></li>
     | ^
  15 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-cypress" target="_blank" rel="noopener">e2e-cypress</a></li>
  16 |     </ul>
  17 |     <h3>Essential Links</h3>


error: Line 15 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:15:1:
  13 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
  14 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-unit-jest" target="_blank" rel="noopener">unit-jest</a></li>
> 15 |       <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-cypress" target="_blank" rel="noopener">e2e-cypress</a></li>
     | ^
  16 |     </ul>
  17 |     <h3>Essential Links</h3>
  18 |     <ul>


error: Line 29 exceeds the maximum line length of 120 (max-len) at src/components/HelloWorld.vue:29:1:
  27 |       <li><a href="https://router.vuejs.org" target="_blank" rel="noopener">vue-router</a></li>
  28 |       <li><a href="https://vuex.vuejs.org" target="_blank" rel="noopener">vuex</a></li>
> 29 |       <li><a href="https://github.com/vuejs/vue-devtools#vue-devtools" target="_blank" rel="noopener">vue-devtools</a></li>
     | ^
  30 |       <li><a href="https://vue-loader.vuejs.org" target="_blank" rel="noopener">vue-loader</a></li>
  31 |       <li><a href="https://github.com/vuejs/awesome-vue" target="_blank" rel="noopener">awesome-vue</a></li>
  32 |     </ul>


6 errors found.

Module Warning (from ./node_modules/eslint-loader/index.js):
error: Missing semicolon (semi) at src/components/HelloI18n.vue:11:56:
   9 | 
  10 | <script lang="ts">
> 11 | import { Component, Vue } from 'vue-property-decorator'
     |                                                        ^
  12 | 
  13 | @Component
  14 | export default class HelloI18n extends Vue {


1 error found.
1 error potentially fixable with the `--fix` option.

You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.                                                                                                                                                                                    Type checking in progress...  

  App running at:
  - Local:   http://localhost:8080/ 
  - Network: http://10.0.0.182:8080/

  Note that the development build is not optimized.
  To create a production build, run npm run build.

No type errors found
Version: typescript 3.4.5
Time: 10798ms
```

Reply:
I did hack this line `responseDeserializeFn: (bytes: Uint8Array) => Response);` just to try it out.

Reply:
I was going to fix this on another branch. But I fixed it anyways. The only lint errors now are `(bytes: Uint8Array)` related.